scripts/prepare.js

```diff
@@ -0,0 +1,36 @@
+const { spawn } = require('child_process')
+
+function runCommand(command, args) {
+  return new Promise((resolve, reject) => {
+    const process = spawn(command, args)
+
+    // Log when the process starts
+    console.log(`${command} ${args} initiated`)
+
+    // Uncomment to see output of each command
+    // process.stdout.on('data', (data) => {
+    //   console.log(`${command}: ${data}`)
+    // })
```

Comment:
Why don't we want to see output? Could you instead condition this on a `--verbose` flag?